services/server/config/config_test.go

```diff
@@ -48,34 +48,34 @@ func TestMergeConfigs(t *testing.T) {
 	}
 
 	err := mergeConfig(a, b)
-	assert.NilError(t, err)
+	assert.Nil(t, err)
 
 	assert.Equal(t, a.Version, 2)
 	assert.Equal(t, a.Root, "new_root")
 	assert.Equal(t, a.State, "old_state")
 	assert.Equal(t, a.OOMScore, 2)
-	assert.DeepEqual(t, a.RequiredPlugins, []string{"old_plugin", "new_plugin1", "new_plugin2"})
-	assert.DeepEqual(t, a.DisabledPlugins, []string{"old_plugin"})
-	assert.DeepEqual(t, a.Timeouts, map[string]string{"a": "1", "b": "2"})
-	assert.DeepEqual(t, a.StreamProcessors, map[string]StreamProcessor{"1": {Path: "3"}, "2": {Path: "5"}})
+	assert.Equal(t, a.RequiredPlugins, []string{"old_plugin", "new_plugin1", "new_plugin2"})
+	assert.Equal(t, a.DisabledPlugins, []string{"old_plugin"})
+	assert.Equal(t, a.Timeouts, map[string]string{"a": "1", "b": "2"})
+	assert.Equal(t, a.StreamProcessors, map[string]StreamProcessor{"1": {Path: "3"}, "2": {Path: "5"}})
```

Comment:
Testify uses the order (expected, actual)

```suggestion
	assert.Equal(t, []string{"old_plugin", "new_plugin1", "new_plugin2"}, a.RequiredPlugins)
	assert.Equal(t, []string{"old_plugin"}, a.DisabledPlugins)
	assert.Equal(t, map[string]string{"a": "1", "b": "2"}, a.Timeouts)
	assert.Equal(t, map[string]StreamProcessor{"1": {Path: "3"}, "2": {Path: "5"}}, a.StreamProcessors)
```